models/tasks.js

```diff
@@ -193,6 +197,22 @@ const fetchPaginatedTasks = async ({
         .where("title", "<=", title + "\uf8ff")
         .orderBy("title", "asc");
     }
+    if (assigneeRole === ROLES.ARCHIVED) {
+      let archivedUserIds = [];
+      let usersData;
+      if (nextArchivedUser && !next) {
+        usersData = await fetchPaginatedArchivedUsers(FIRESTORE_IN_CLAUSE_SIZE, nextArchivedUser, "next");
+      }
+      if (!nextArchivedUser && !next) {
+        usersData = await fetchPaginatedArchivedUsers(FIRESTORE_IN_CLAUSE_SIZE);
+      }
+      if (nextArchivedUser && next) {
+        usersData = await fetchPaginatedArchivedUsers(FIRESTORE_IN_CLAUSE_SIZE, nextArchivedUser, "prev");
+      }
```

Comment:
Can you please explain what you are trying to do here?

Reply:
What will happen if I pass size as 100 with assigneeRole as archived? Will I get 100 tasks? or only the tasks assigned to 30 archived userIds?

Reply:
It will be only tasks assigned to 30 archived userIds, if the tasks assigned to current batch of 30 users is 100 or more you will be getting 100 tasks and the cursor to the next if it exists.

Reply:
Not sure if we should break existing contracts for this. @prakashchoudhary07 @yesyash @heyrandhir what's your opinion on this?